index.ts

```diff
@@ -51,6 +52,26 @@ app.post(
     }
 )
 
+app.post(
+    "/searchItemName",
+    async (
+        req: Request<unknown, unknown, itemStuff, unknown>,
+        res: Response
+    ) => {
+        try{
+            const response = await searchItemName(req.body.itemName);
+            if(response == 200){
+                const token = jwt.sign(req.body.itemName, "memes");
```

Comment:
Why are you issuing a jwt on item search?

Reply:
Left behind from testing, it has been removed.